examples/api/photo-stream.js

```diff
@@ -17,6 +17,9 @@
 
 const express = require('express');
 const {setNoCache} = require('@lib/utils/cacheHelpers');
+const {LoremIpsum} = require('lorem-ipsum');
```

Comment:
I't be cool to have something that generates random text (which seems to make sense on the first look).

Reply:
I looked into https://www.npmjs.com/package/casual and https://www.npmjs.com/package/faker but it seems no different than https://www.npmjs.com/package/lorem-ipsum when it comes to descriptions.

Happy to replace later if we find a better random text provider. :)